components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -589,14 +570,8 @@ void RewardsServiceImpl::OnLoad(SessionID tab_id, const GURL& url) {
 
   const std::string publisher_url = origin.scheme() + "://" + baseDomain + "/";
 
-  ledger::VisitData data(baseDomain,
-                         origin.host(),
-                         url.path(),
-                         tab_id.id(),
-                         baseDomain,
-                         publisher_url,
-                         "",
-                         "");
+  ledger::VisitData data(baseDomain, origin.host(), url.path(), tab_id.id(),
```

Comment:
same as above. I think this is much harder to read and to see which params are which

Reply:
Latest patch does not touch this file, which sidesteps these concerns. (I still support a repository-wide enforced style based on `clang-format`, but it's now decoupled from this PR.)